modules/ROOT/pages/install-prereqs.adoc

```diff
@@ -0,0 +1,404 @@
+See xref:architecture.adoc[Runtime Fabric Architecture] for a description of development and production configurations.
+
+== General Knowledge
+Knowledge in the following areas is required to perform a Runtime Fabric installation:
+
+* Linux system commands
+* Working in Linux with root access. Installation requires root access (permission to run the `sudo` command) on all servers in your environment
+* If running on AWS, working with Terraform
+* Remote connection mechanisms, such as SSL and SSH
+
+== General Considerations
+Ensure you plan for:
+
+* Creating separate Runtime Fabrics to separate production and non-production applications
+* Creating separate Runtime Fabrics based upon environment network scope, such as DMZ versus private network 
+* The Runtime Fabric default pod and services CIDR block not interfering with the regions network
+
+== Infrastructure Requirements
+Perform the following actions:
+
+* Disable swap memory.
+* Ensure chrony (system clocks) status is in sync.
+* Provision the required hardware, or have enough quota with your infrastructure provider to create the required hardware. 
++
+This includes:
+
+** x86 or x64 architecture. Only x86 and x64 architectures are supported.
+** Virtual machines
+** Disks with provisioned IOPS
+** Virtual networks
+** Firewall rules / security groups
+** Load balancers
+* If provisioning hardware with AWS or Azure, you must have the right permissions to create the required resources.
++
+[NOTE]
+Dedicated disks are required to ensure optimum performance. The Runtime Fabric installer formats
+and mounts the disks needed during installation.
++
+The default behavior of the AWS and Azure provisioning scripts creates a set of virtual machines and disks defined by 
+the minimum requirements. It also creates a private network with the required network ports configured. This is optimal 
+when evaluating Runtime Fabric before integrating within your primary network. You should consult with your 
+network administrator to determine if the default behavior is compatible. You might need to modify the 
+provisioning scripts to accommodate your organization's requirements.
+* Gain an understanding of how to forward logs to your organization's logging provider.
+* Obtain your organization's SMTP provider information so that you can configure alerts.
+
+* Verify the following items for your network:
+
+** Outbound connections are routed to a proxy.
+** Outbound access is allowed using websockets (via port 443) to Anypoint Platform.
+** Your organization's network and security policies, including:
+
+*** Internal network IP address range in CIDR notation
+*** The subnet range used for Runtime Fabric in CIDR notation
+*** If virtual machines are allowed, public IP addresses
+*** How to gain shell access to each VM
+** Determine if inbound traffic should be enabled for Anypoint Runtime Fabric. This is based on the function of your organization's Mule applications. To enable inbound traffic, you must:
```

Comment:
So, again, the list now departs into a subtopic of HOW to meet a requirement, rather than just listing a requirement.
I know that this is an effect of your attempt to meet audience requests to combine previously separate topics, but I think that this effort might require you to omit or move the "How to" instructions (separate Task topic) and the "Why" explanations (Concepts) from this "Requirements" list (Reference).

Reply:
What this means is that we really should omit "To enable inbound traffic, you must:" and the steps following, so that this subtopic about Infrastructure preparation or requirements ends with inbound traffic being enabled, if necessary. Period.